manage.py

```diff
@@ -51,19 +46,14 @@ def test(all=False):
 @manager.command
 def scrape():
     """Start an infinte loop to scrape & categorize articles."""
-    iterations = 0
     categorizer = Categorizer()
 
     while True:
         logging.info("Starting scrape...")
-        if (iterations % ITERATIONS_BEFORE_RESET) == 0:
-            categorizer.reset_title_categories()
 
         scrape_articles(categorizer)
 
-        iterations += 1
-        logging.info("Finished scrape. Zzz...")
-        time.sleep(SCRAPE_INTERVAL)
+        logging.info("Finished scrape. Let's run it back...")
```

Comment:
We don't exactly need "sleep" but more like "pass control on to other functions for a while and don't waste your computation energy on the scraper for now". Aka redis queue...